LPS-68025 Fix id in POSHI test

ambrinchaudhary opened the pull request:
Hey @sharonchoi,
Could you please help me to find out why the POSHI test are failing? I tried to fix on test (last commit) but did not work :-(, I do not know why is exactly failing, because I did not change that behaviour 
Many thanks in advance!

ambrinchaudhary committed:
LPS-68025 Enable drag&drop for existing pages

ambrinchaudhary committed:
LPS-68025 Set upload url for wiki page attachments

ambrinchaudhary committed:
LPS-68025 Allow to set a rootElement to define the upload area

ambrinchaudhary committed:
LPS-68025 Get the proper image src depending on the editor format

ambrinchaudhary committed:
LPS-68025 Rename plugin

Commit:
LPS-68025 SF

ambrinchaudhary committed:
LPS-68025 Create WikiPorlet as metal component and move button actions to it.

ambrinchaudhary committed:
LPS-68025 Check if there are temporal images before saving the page

ambrinchaudhary committed:
LPS-68025 Move the select change event to the js module

ambrinchaudhary committed:
LPS-68025 Add JsDoc

sherlyliu committed:
LPS-68687 Shouldn't add any record with live goup id when we operate staging UI.

ambrinchaudhary committed:
LPS-68025 Make PortletBase extends from Component and simplify WikiPortlet

ambrinchaudhary committed:
LPS-68025 Update doc

daledotshan committed:
LPS-68687 only not add calendar for live group

natocesarrego committed:
LPS-68687 - Move variable to be closer from where it'll be used.

ambrinchaudhary committed:
LPS-68025 Gulp tasks

Commit:
LPS-38562 Hide non needed empty message

ksuetani committed:
LRQA-26989 Add new test to verify System Settings inheritance to Virtual Instance

Commit:
SYNC-1696 After syncdlobject exist (related resourcepermission also exist), if we give one new role (at this time, resourcepermission doesn't exist related this role data) assign view permission, we also should update syncdlobject's modifiedTime. So that it can be returned related data from syncdlobject table when sync executes update request.

ambrinchaudhary committed:
LPS-68025 Update metal-cli dependency

ambrinchaudhary committed:
LPS-68025 Do not introduce breaking changes

ambrinchaudhary committed:
LPS-68025 Move to frontend-editor-alloyeditor-web module as it is used only in AlloyEditor

ambrinchaudhary committed:
LPS-68025 Save_() method should check if there are temporal images

daledotshan committed:
SYNC-1696 SF

ambrinchaudhary committed:
LPS-68025 Update test: 'ae_addimages' plugin is enabled when wiki page exists

shinnlok committed:
SYNC-1696 SF

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/877/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/877/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED.</h1><p>Build Time: 3 hours 25 minutes 44 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/30021504e07312cff9256abddc5e1604294a198e">30021504e07312cff9256abddc5e1604294a198e</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/877/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57742/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60966/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58585/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60968/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54869/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat8-oracle12-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54870/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60969/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat8-sybase16-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60972/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58588/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57745/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43209/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43248/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13868/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/10</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43210/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/11</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58592/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/12</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39471/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/13</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59975/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/14</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13862/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/15</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57748/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/16</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39472/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/17</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58594/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/18</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/56677/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/19</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58591/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13869/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/20</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57746/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/21</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43251/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/22</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43213/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/23</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43214/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/24</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54872/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/25</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/26</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59978/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/27</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53351/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/28</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58013/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/29</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13867/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12746/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/30</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60974/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/31</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50646/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/32</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58697/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/33</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54873/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/34</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53352/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/35</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43215/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/36</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58014/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/37</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54874/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/38</a></li><li><strike><strong><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50655/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/39</a></strong></strike></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39470/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/4</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53012/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/40</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12747/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/41</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53353/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/42</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59117/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/5</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43249/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/6</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13861/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58593/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57747/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/9</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60977/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58016/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-mariadb100-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60978/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/57749/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50650/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-oracle12-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58017/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-oracle12-jdk8/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39473/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50652/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat8-postgresql94-jdk8/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54878/">test-portal-acceptance-pullrequest-batch(master)/integration-db2105-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58018/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/56678/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50651/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle12-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39474/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13870/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase160-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43219/">test-portal-acceptance-pullrequest-batch(master)/lpkg-override-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58702/">test-portal-acceptance-pullrequest-batch(master)/lpkg-persistence-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43218/">test-portal-acceptance-pullrequest-batch(master)/lpkg-release-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59982/">test-portal-acceptance-pullrequest-batch(master)/lpkg-startup-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54879/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><strike><strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58701/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</a></strong></strike></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43220/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60981/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/53013/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle12-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54881/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50653/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/39475/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43253/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59985/">test-portal-acceptance-pullrequest-batch(master)/portal-frontend-js-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13872/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12748/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59984/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13871/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/59986/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58019/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43255/">test-portal-acceptance-pullrequest-batch(master)/wsdd-builder-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/458/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/955/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/877/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/877/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>88 Jobs Passed.<br />3 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/50655/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/39</a></h5><h6>Job Results:</h6><p>1 Test Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/50655//testReport/com.liferay.poshi.runner/PoshiRunner/test_PGWiki_AddWikiFrontPageChildPageFormatHTML_">PGWiki#AddWikiFrontPageChildPageFormatHTML</a> - <a href="https://testray.liferay.com/reports/production//logs/test-1-2/1479291603979/test-portal-acceptance-pullrequest(master)/877/functional-tomcat8-mysql56-jdk8/39/0/PGWiki_AddWikiFrontPageChildPageFormatHTML/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production//logs/test-1-2/1479291603979/test-portal-acceptance-pullrequest(master)/877/functional-tomcat8-mysql56-jdk8/39/0/PGWiki_AddWikiFrontPageChildPageFormatHTML/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production//logs/test-1-2/1479291603979/test-portal-acceptance-pullrequest(master)/877/functional-tomcat8-mysql56-jdk8/39/0/jenkins-console.txt.gz">Console Output</a></li></ol></li><li><h5 job-result="FAILURE"><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/58701/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</a></h5><h6>Job Results:</h6><p>14 Tests Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=6,label_exp=!master/58701/">test-portal-acceptance-pullrequest-batch&#40;master&#41; ?? 6,!master #58701</a></strong><pre><code>     [exec] 12:13:21,259 INFO  [localhost-startStop-1][BundleStartStopLogger:35] STARTED com.liferay.announcements.web_1.1.0 [536]
     [exec] 12:13:21,342 INFO  [localhost-startStop-1][BundleStartStopLogger:35] STARTED com.liferay.social.requests.web_1.0.4 [539]
     [exec] 12:13:21,389 INFO  [localhost-startStop-1][BundleStartStopLogger:35] STARTED com.liferay.user.groups.admin.impl_1.0.5 [541]
     [exec] 12:13:21,843 INFO  [fileinstall-/opt/dev/projects/github/liferay-portal/bundles/osgi/modules][BundleStartStopLogger:35] STARTED com.liferay.message.boards.web_1.2.2 [508]
     [exec] 12:13:21,895 INFO  [fileinstall-/opt/dev/projects/github/liferay-portal/bundles/osgi/modules][BundleStartStopLogger:35] STARTED com.liferay.journal.content.asset.addon.entry.common_2.0.4 [554]
     [exec] 12:13:22,258 INFO  [fileinstall-/opt/dev/projects/github/liferay-portal/bundles/osgi/modules][BundleStartStopLogger:35] STARTED com.liferay.shopping.web_1.0.9 [557]
     [exec] 12:13:22,364 INFO  [localhost-startStop-1][BundleStartStopLogger:35] STARTED com.liferay.wiki.service_1.2.15 [542]
     [exec] 16-Nov-2016 12:13:22.378 INFO [localhost-startStop-1] org.apache.catalina.core.ApplicationContext.log Initializing Spring root WebApplicationContext
     [exec] :test:arquillian-extension-junit-bridge-test:startTestableTomcat FAILED
     [exec] :test:arquillian-extension-junit-bridge-test:stopTestableTomcat
     [exec] 
     [exec] BUILD FAILED
     [exec] 
     [exec] Total time: 33 mins 4.588 secs
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':test:arquillian-extension-junit-bridge-test:startTestableTomcat'.
     [exec] > Timeout while waiting for the application server
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':test:arquillian-extension-junit-bridge-test:startTestableTomcat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:61)</code></pre></li></ol></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

Preston-Crary committed:
LPS-69000 Use the same tika config as FileImpl rather than trying to recreate it.

Preston-Crary committed:
LPS-68289 Remove IdempotentRetryAssert usages from where it is not needed, we have liferay.mode=true and SynchronousDestinationTestRule to force synchronous behavior for search integration tests.

Preston-Crary committed:
LPS-68289 Implement toString() in HitsImpl, toString() is already implemented in DocumentImpl.

HEAD is now at b0d4c8b LPS-68289 Implement toString() in HitsImpl, DocumentImpl already has toString() implemented so now we can get a text view of a search result.

Preston-Crary committed:
LPS-68289 Improve search test assertions when comparing hits lengths.

ksuetani committed:
LRQA-28911 Add macro to assert cluster name in module

ksuetani committed:
LRQA-28911 Add macro to edit a text area in System Settings

ksuetani committed:
LRQA-28911 Add test to assert System Settings propogate to Elasticsearch module

tinatian committed:
LPS-68289 Should use SynchronousDestinationTestRule

kenjiheigel committed:
LPS-66646 Update naming

Commit:
LRQA-28911 Make path less precise

Commit:
LRQA-28911 Type full setting value

Commit:
LRQA-28911 Assert full line of the cluster status

Commit:
LRQA-28911 Don’t use portlet-specific locator

sharonchoi commented:
Just started reviewing :)

pat270 committed:
LPS-69290 - Forms and Workflow Info Panels should use correct navbar

brianwulbern committed:
LRQA-28794 Add time before saving Edit

marcellustavares committed:
LPS-69290 Use a released version

sharonchoi commented:
@ambrinchaudhary So it turns out the issue is that, when a user tries to select the page format before entering the title of the wiki, the portal will throw the error messages:

![ezgif com-video-to-gif 2](https://cloud.githubusercontent.com/assets/2567789/20439072/d833e82a-ad6e-11e6-8c59-71b23c4eb437.gif)

brandizzi committed:
LPS-68687 Tests for CalendarResourceUtil (including current issue)

brandizzi committed:
LPS-68687 Prevent live site calendar creation all the times (not only when adding to "Other calendars").

marcellustavares committed:
LPS-68687 SF

alee8888 committed:
LRQA-26989 SF

Ithildir committed:
LPS-68289 Print timestamp when Gradle build finishes

brandizzi committed:
LPS-68687 Remove use of updated method from test which depended on old behavior.

brandizzi committed:
LPS-38562 Updating POSHI test - message is not displayed anymore.

brianchandotcom committed:
LPS-68072

pat270 committed:
LPS-69313 - Management Bar Mobile Toggle should use Lexicon icon

brianchandotcom committed:
LRQA-28693 Ivy needs mirrors-get

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

pat270 committed:
LPS-69305 - Portlet Topper should use Lexicon SVG Icon

michaelhashimoto committed:
LRQA-29213 Mark as a CE test environment

shuyangzhou committed:
LPS-69000 Simplify

michaelhashimoto committed:
LRQA-29213 Mark EE only tests

tomwang2011 committed:
LPS-68289 Handle Document library sync event processor synchronously

brianchandotcom committed:
Revert "portal-kernel 2.15.0 cache"

This reverts commit ed397e912b4f3594ebe3b80e39a828cc0d5d8cd8.

shuyangzhou committed:
LPS-68289 Check Arquillian at class level

brianchandotcom committed:
portal-kernel 2.15.0 proper cache

brianchandotcom committed:
portal-kernel prep com.liferay.portal.kernel.util 7.12.0 package

brianchandotcom committed:
portal-kernel prep to 2.16.0, Andrea, baseline didn't force me to do this, I think it's a bug?

brianchandotcom committed:
artifact:ignore chat-api 1.0.2 prep next

brianchandotcom committed:
artifact:ignore chat-service 1.0.7 prep next

brianchandotcom committed:
artifact:ignore chat-web 1.0.4 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-service 1.1.12 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-web 1.0.22 prep next

brianchandotcom committed:
artifact:ignore mail-reader-service 2.0.6 prep next

brianchandotcom committed:
artifact:ignore marketplace-service 2.1.18 prep next

brianchandotcom committed:
artifact:ignore opensocial-portlet 7.0.0.9 prep next

brianchandotcom committed:
artifact:ignore portal-scripting-javascript 2.0.5 prep next

brianchandotcom committed:
artifact:ignore push-notifications-service 1.0.8 prep next

brianchandotcom committed:
artifact:ignore shopping-service 2.0.15 prep next

brianchandotcom committed:
artifact:ignore social-networking-service 2.0.9 prep next

brianchandotcom committed:
artifact:ignore social-private-messaging-service 2.0.0 prep next

brianchandotcom committed:
social-private-messaging-service 2.0.0 apply

brianchandotcom committed:
artifact:ignore sync-service 1.3.4 prep next

brianchandotcom committed:
artifact:ignore wsrp-service 1.0.10 prep next

brianchandotcom committed:
artifact:ignore wysiwyg-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore wysiwyg-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore bookmarks-service 1.1.2 prep next

brianchandotcom committed:
artifact:ignore comment-web 1.1.3 prep next

brianchandotcom committed:
artifact:ignore invitation-invite-members-service 2.0.10 prep next

brianchandotcom committed:
artifact:ignore mentions-web 1.1.3 prep next

brianchandotcom committed:
artifact:ignore message-boards-web 1.2.2 prep next

brianchandotcom committed:
artifact:ignore microblogs-service 2.1.5 prep next

brianchandotcom committed:
artifact:ignore microblogs-web 2.0.9 prep next

brianchandotcom committed:
artifact:ignore wiki-service 1.2.15 prep next

brianchandotcom committed:
wiki-service 1.2.15 apply

brianchandotcom committed:
artifact:ignore wiki-web 2.1.10 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-type-options 2.0.7 prep next

brianchandotcom committed:
artifact:ignore polls-service 3.0.11 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-service 2.0.15 prep next

brianchandotcom committed:
artifact:ignore contacts-service 2.0.7 prep next

brianchandotcom committed:
artifact:ignore mobile-device-rules-service 1.2.6 prep next

brianchandotcom committed:
artifact:ignore portal-cache 2.2.1 prep next

brianchandotcom committed:
artifact:ignore portal-lock-service 2.0.7 prep next

brianchandotcom committed:
artifact:ignore portal-search 3.4.6 prep next

brianchandotcom committed:
artifact:ignore portal-search-elasticsearch 2.0.11 prep next

brianchandotcom committed:
artifact:ignore portal-security-antisamy 2.0.5 prep next

brianchandotcom committed:
artifact:ignore portal-security-service-access-policy-service 2.0.12 prep next

brianchandotcom committed:
artifact:ignore portal-monitoring 4.0.10 prep next

brianchandotcom committed:
artifact:ignore portal-verify-extender 2.1.2 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-wab-generator 2.1.14 prep next

brianchandotcom committed:
artifact:ignore export-import-web 1.0.16 prep next

brianchandotcom committed:
artifact:ignore product-navigation-control-menu-web 2.0.8 prep next

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 2.0.7 prep next

brianchandotcom committed:
artifact:ignore staging-processes-web 1.0.10 prep next

brianchandotcom committed:
artifact:ignore staging-security 2.0.14 prep next

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/journal/.gitrepo'.

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore social-private-messaging-web 1.0.14 prep next

brianchandotcom committed:
artifact:ignore wiki-editor-configuration 1.0.6 prep next

brianchandotcom committed:
modules/core/portal-bootstrap/system.packages.extra.mf regen

brianchandotcom committed:
artifact:ignore frontend-theme-fjord 1.0.7 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-porygon 1.0.3 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-westeros-bank 1.0.18 prep next

brianchandotcom committed:
artifact:ignore twitter-api 1.0.0 prep next

brianchandotcom committed:
twitter-api 1.0.0 packageinfo

brianchandotcom committed:
artifact:ignore twitter-api 1.0.0 prep next

brianchandotcom committed:
twitter-api 1.0.0 apply

brianchandotcom committed:
artifact:ignore frontend-theme-admin 1.0.16 prep next

brianchandotcom committed:
LPS-68687 optimize for humans

brianchandotcom committed:
artifact:ignore frontend-theme-classic 1.0.18 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-styled 2.0.15 prep next

brianchandotcom committed:
LPS-68687 SF

brianchandotcom committed:
artifact:ignore frontend-theme-user-dashboard 1.0.16 prep next

brianchandotcom committed:
LPS-68687 SF

brianchandotcom committed:
artifact:ignore frontend-theme-user-profile 1.0.16 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore twitter-service 1.0.0 prep next

brianchandotcom committed:
twitter-service 1.0.0 packageinfo

brianchandotcom committed:
artifact:ignore twitter-service 1.0.0 prep next

brianchandotcom committed:
twitter-service 1.0.0 apply

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore twitter-web 1.0.0 prep next

brianchandotcom committed:
artifact:ignore twitter-web 1.0.0 prep next

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/bookmarks/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/comment/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/invitation/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/mentions/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/microblogs/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/wiki/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/calendar/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-mapping/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/polls/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/portal-workflow/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/contacts/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-taglib/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/mobile-device-rules/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-cache/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-lock/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-security/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

ambrinchaudhary committed:
LPS-68025 Do not save the page when the format is changed

ambrinchaudhary committed:
LPS-68025 Update package version and add shrinkwrap file

ambrinchaudhary commented:
Thank you @sharonchoi for helping me find the error. I'm closing the PR
